assets/airports/essa.json

```diff
@@ -0,0 +1,2865 @@
+            "draw": [
+                ["SA871", "SA861", "NOSLI"]
+            ]
+        },
+        "PETEV1K": {
+            "icao": "PETEV1K",
+            "name": "Petev One Kilo",
+            "altitude": 5000,
+            "rwy": {
+                "ESSA26": []
+            },
+            "body": [["^SA871", "A05+"], "SA861", "PETEV"],
+            "exitPoints": {
+                "PETEV": ["PETEV"]
+            },
+            "draw": [
+                ["SA871", "SA861", "PETEV"]
+            ]
+        },
+        "RESNA3K": {
+            "icao": "RESNA3K",
+            "name": "Resna Three Kilo",
+            "altitude": 5000,
+            "rwy": {
+                "ESSA26": []
+            },
+            "body": [["^SA871", "A06+"], ["SA862", "A15+|S220-"], "RESNA"],
+            "exitPoints": {
+                "RESNA": ["RESNA"]
+            },
+            "draw": [
+                ["SA871", "SA862", "RESNA"]
+            ]
+        },
+        "TOVRI4K": {
+            "icao": "TOVRI4K",
+            "name": "Tovri Four Kilo",
+            "altitude": 5000,
+            "rwy": {
+                "ESSA26": []
+            },
+            "body": [["^SA871", "A06+"], ["SA419", "A15+|S22-"], "TOVRI"],
+            "exitPoints": {
+                "TOVRI": ["TOVRI"]
+            },
+            "draw": [
+                ["SA871", "SA419", "TOVRI"]
+            ]
+        },
+        "TROSA3K": {
+            "icao": "TROSA3K",
+            "name": "Trosa Three Kilo",
+            "altitude": 5000,
+            "rwy": {
+                "ESSA26": []
+            },
+            "body": [["^SA871", "A05+"], "SA861", "SA719", "TRS"],
+            "exitPoints": {
+                "TRS": ["TRS"]
+            },
+            "draw": [
+                ["SA871", "SA861", "SA719", "TRS"]
+            ]
+        }
+    },
+    "_comment": "Source runway 01L STARs: https://www.aro.lfv.se/Editorial/View/7949/ES_AD_2_ESSA_4-47_en",
+    "_comment": "Source runway 01L/01R STARs: https://www.aro.lfv.se/Editorial/View/6273/ES_AD_2_ESSA_4-51_en",
+    "_comment": "Source runway 08 STARs: https://www.aro.lfv.se/Editorial/View/6274/ES_AD_2_ESSA_4-53_en",
+    "_comment": "Source runway 19R STARs: https://www.aro.lfv.se/Editorial/View/7950/ES_AD_2_ESSA_4-55_en",
+    "_comment": "Source runway 19R/19L STARs: https://www.aro.lfv.se/Editorial/View/6276/ES_AD_2_ESSA_4-59_en",
+    "_comment": "Source runway 26 STARs: https://www.aro.lfv.se/Editorial/View/7582/ES_AD_2_ESSA_4-61_en",
+    "_comment": "Source runway 26 open STARs: https://www.aro.lfv.se/Editorial/View/6278/ES_AD_2_ESSA_4-65_en",
+    "stars": {
+        "ELTOK3J": {
+            "icao": "ELTOK3J",
+            "name": "Eltok Three Juliet",
+            "entryPoints": {
+                "ELTOK": ["ELTOK"]
+            },
+            "body": ["ELTOK", ["SA912", "A100+"], ["SA810", "A80+"], "SA815", ["SA808", "A50+"], ["SA816", "A25+|S200-"], ["SA802", "A25+|S180-"], ["SA801", "A25+"]],
```

Comment:
As my comment regarding the SIDs, I guess that Eltok maybe doesn't need to be both an entryPoint and the first point in the body? I'll do the same for all STARs for now and then change them all after feedback.

Reply:
I removed duplicates from the beginning of the body so they are only in entryPoints. Will push!